build/docker/structure-test.yaml

```diff
@@ -14,8 +14,7 @@ commandTests:
   - name: "journalctl is working in the image"
     command: "journalctl"
     expectedOutput: ["-- No entries --\n"]
-    excludedError: ["No journal files were found.\n"]
   - name: "swi-otelcol is working in the image"
     command: "/swi-otelcol"
     args: ["-v"]
-    expectedOutput: ["swi-k8s-opentelemetry-collector version 0.8.4"]
\ No newline at end of file
+    expectedOutput: ["swi-k8s-opentelemetry-collector version 0.8.5"]
```

Comment:
This was forgotten in one of the previous commits and is not directly related to the goal of this PR.